### ISSUE TITLE
Build failure: yubioath-flutter

### ISSUE THREAD

**Issue by ElnuDev** [NixOS/nixpkgs#232092]
### Steps To Reproduce

1. build `yubioath-flutter`

### Build log

```
error: builder for '/nix/store/jalgglf19k4zfh6c5yv18kmkxw6xlw6q-yubioath-flutter-helper-6.1.0.drv' failed with exit code 1;
       last 10 log lines:
       > installing
       > Executing pipInstallPhase
       > /build/source/helper/dist /build/source/helper
       > Processing ./authenticator_helper-0.1.0-py3-none-any.whl
       > Requirement already satisfied: zxing-cpp in /nix/store/dxnln5rmjs87qw1sd34vgjj8b5iqc7mg-python3.10-zxing_cpp-1.4.0/lib/python3.10/site-packages (from authenticator-helper==0.1.0) (1.4.0)
       > Requirement already satisfied: Pillow<10.0.0,>=9.1.1 in /nix/store/49y0gryvz22pfjhwq6x0gbawp2h7vsaa-python3.10-pillow-9.4.0/lib/python3.10/site-packages (from authenticator-helper==0.1.0) (9.4.0)
       > Requirement already satisfied: mss in /nix/store/xq1nygacq8mc5rg53qp6ccs0pcgm8pid-python3.10-mss-7.0.1/lib/python3.10/site-packages (from authenticator-helper==0.1.0) (7.0.1)
       > ERROR: Could not find a version that satisfies the requirement yubikey-manager==5.0.1 (from authenticator-helper) (from versions: none)
       > ERROR: No matching distribution found for yubikey-manager==5.0.1
       > 
       For full logs, run 'nix log /nix/store/jalgglf19k4zfh6c5yv18kmkxw6xlw6q-yubioath-flutter-helper-6.1.0.drv'.
```

### Additional context

I got this issue after upgrading my system from 897876e4c484f1e8f92009fd11b7d988a121a4e7 to 0470f36b02ef01d4f43c641bbf07020bcab71bf1, it seems ff9c88829755b38c3d44cd3b7446e389ce669697 broke yubioath-flutter somehow.

### Notify maintainers

@lukegb @hacker1024 

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[elnu@elnu:~/nix-config]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.3.1, NixOS, 23.05 (Stoat), 23.05.20230506.897876e`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.13.3`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by hacker1024** [comment]
Closed by #230674.

**Comment by Aleksanaa** [comment]
```
❯ nix run nixpkgs#nix-info -- -m
 - system: `"x86_64-linux"`
 - host os: `Linux 6.3.1-zen1, NixOS, 23.05 (Stoat), 23.05.20230522.7084250`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.13.3`
 - nixpkgs: `/nix/store/hc2y6044ksiz4vs1cyhy6qzd6szxvm4c-source`
```
Builds and runs correctly for me.
